docs/narr/project.rst

```diff
@@ -871,6 +869,8 @@ also informs Python that the directory which contains it is a *package*.
    Line 11 returns a :term:`WSGI` application to the caller of the function
    (Pyramid's pserve).
 
+   See :ref:`the_startup_process` for more about the application's :ref:`settings <startup_settings>` and ``main``\'s :ref:`arguments <startup_constructor_arguments>` and statements.
```

Comment:
Let's make this a [`.. seealso::`](https://docs.pylonsproject.org/projects/docs-style-guide/#see-also) directive so it stands out from the flow of text.